samples/excel/tables/add-rows-to-a-table.yaml

```diff
@@ -0,0 +1,164 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Add rows to a table
+description: Add rows to a table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#add-rows').click(addRows);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
+            let sheet: Excel.Worksheet;
+            try {
+                // If sheet already exists, queue a command to delete it
```

Comment:
I want you to consider having a Setup action/function in the snippet similar to the one in Range.yaml to basically do a reset of the current worksheet.